shared/react-native/packageAndBuild.sh

```diff
@@ -46,5 +46,5 @@ sleep 5 && requestbundles &
 backgroundpid=$!
 trap 'kill $backgroundpid' EXIT # quit requestBundles on exit
 while true; do
-  yarn react-native start
+  yarn react-native start 
```

Comment:
Did you need that space?

Reply:
my ide used to delete those on save and that broke, fixed